src/99-appendices/03-hed.md

```diff
@@ -75,5 +75,42 @@ The tags in the `HED` column are often specific to the event instances, while
 the common properties associated with categories such as `mycodes` are
 encapsulated in the \_events.json dictionary. Downstream tools should not
 distinguish between tags specified using the different mechanisms. Further,
-the normal BIDS inheritance principle applies so these data dictionaries can
+the normal BIDS inheritance principle applies, so these data dictionaries can
 appear higher in the BIDS hierarchy.
+
+The HED vocabulary is specified by a HED schema, which delineates the allowed HED path strings. By default, 
+BIDS uses the latest HED schema available in the
+[hed-specification](https://github.com/hed-standard/hed-specification/tree/master/hedxml) repository
+maintained by the hed-standard group. 
+
+You can override the default by providing a specific HED version number or including a HED XML file
+in the `sourcedata` directory at the top level of the study. You would indicate the version or
+the XML filename using the optional `HEDVersion` field of the `dataset_description.json` file. 
+The preferred method is to validate with the latest version, but to use the `HEDVersion` field to specify
+which version was used for later reference.  
```

Comment:
Some wording suggestions:

Please also note that we try to insert a line break after the end of each sentence. --> this "style" makes the future git diff more readable.

```suggestion
Using the latest available HED schema (the default in BIDS) is RECOMMENDED.
However, you can override this default by providing a specific HED version number or by including a HED XML file
in the `sourcedata` directory at the top level of the study.
The HED version number or the path to the HED XML file MUST be specified using the
recommended `HEDVersion` field of the `dataset_description.json` file. 
```